lib/EnzymeCore/test/mode_modification.jl

```diff
@@ -0,0 +1,15 @@
+using EnzymeCore
+using EnzymeCore: split_mode, unsplit_mode
+using Test
+
+@testset "Split / unsplit mode" begin
+    @test split_mode(Reverse) == ReverseSplitNoPrimal
+    @test split_mode(ReverseWithPrimal) == ReverseSplitWithPrimal
+    @test split_mode(ReverseSplitNoPrimal) == ReverseSplitNoPrimal
+    @test split_mode(ReverseSplitWithPrimal) == ReverseSplitWithPrimal
+
+    @test unsplit_mode(Reverse) == Reverse
+    @test unsplit_mode(ReverseWithPrimal) == ReverseWithPrimal
```

Comment:
Can you also test the use of runtime activity and err if find written

Reply:
done